src/audio/lib/heap.c

```diff
@@ -1109,7 +1109,9 @@ void* AudioHeap_AllocPermanent(s32 tableType, s32 id, size_t size) {
     gAudioCtx.permanentEntries[index].size = size;
     //! @bug UB: missing return. "addr" is in v0 at this point, but doing an
```

Comment:
Could you label this where the UB occurs too? It's not going to be obvious just from this since it occurs when the missing return is used.